suite/runsuite_wrapper.pl

```diff
@@ -86,7 +86,7 @@
         $mydir = `/usr/bin/cygpath -wi \"$mydir\"`;
         chomp $mydir;
     }
-    system("ctest --output-on-failure ${verbose} -S \"${mydir}/runsuite.cmake${args}\" 2>&1");
+    system("ctest -VV --output-on-failure ${verbose} -S \"${mydir}/runsuite.cmake${args}\" 2>&1");
```

Comment:
Please undo the temporary change (suggestion: mark temporary changes with #TEMPORARY or sthg to make it harder to accidentally leave them in).